common/lib/xmodule/xmodule/x_module.py

```diff
@@ -200,10 +200,8 @@ def get_javascript(cls):
         coffee = cls.js.setdefault('coffee', [])
         js = cls.js.setdefault('js', [])
 
-        fragment = resource_string(__name__, 'js/src/xmodule.js')
-
-        if fragment not in js:
-            js.insert(0, fragment)
+        # Added xmodule.js separately to enforce 000 prefix for this only.
+        xmodule_js = cls.js.setdefault('xmodule_js', resource_string(__name__, 'js/src/xmodule.js'))
```

Comment:
In `_write_js`, you insert `xmodule.js` at the front of the list. Why do you need to add it here, too?